### ISSUE TITLE
[SF][A11y][ObjectStatus]: Color contrast is failing for Inverted Object Status

### ISSUE THREAD

**Issue by saorabhkr** [SAP/ui5-webcomponents-react#6154]
### Describe the bug

The Object Status has an inverted visualization, used when the information is crucial for the user’s actions and needs to stand out visually.

While checking it for color contrast, I found it did not match WCAG 2.1 guidelines.

Please refer to the attached screenshots:

### ValueState.Error

<img width="1717" alt="Screenshot 2024-08-04 at 7 21 01 AM" src="https://github.com/user-attachments/assets/067c8a77-c4c8-49b6-b22b-e74cbb25943d">

### ValueState.Warning
<img width="1705" alt="Screenshot 2024-08-04 at 7 21 16 AM" src="https://github.com/user-attachments/assets/5978a9df-199f-42cc-8f91-c939154b6629">

### ValueState.Success
<img width="1671" alt="Screenshot 2024-08-04 at 7 20 46 AM" src="https://github.com/user-attachments/assets/9dfafd2d-602b-4099-8fcb-4dfc32d77e65">

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-objectstatus--docs

### Reproduction steps

It can be observed on Storybook with a Color contrast analyzer. 

### Expected Behaviour

Minimum contrast requirement (AA) for regular text. 

> The required contrast ratio is at least 4.5:1.

Minimum contrast requirement (AA) for large text. 

> Large text needs a minimum contrast ratio of 3:1.

Contrast requirement (AAA) for both regular and large text. 

> The required contrast ratio is at least 7:1 for regular text and 4.5:1 for large text.

### UI5 Web Components for React Version

v1.24.x

### UI5 Web Components Version

1.21.x

### Browser

Chrome, Edge, Firefox, Safari

### Operating System

Mac OS

### Organization

SFSF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @saorabhkr 

the `ObjectStatus` is implemented according to central design and accessibility specifications. The `text-shadow` should ensure the correct contrast ratio. If this is not the case for individual states, then please add them to this [StackBlitz](https://stackblitz.com/github/SAP/ui5-webcomponents-react/tree/main/templates/vite-ts?file=src%2FApp.tsx) template. We will then check if we missed some specifications.
If you think there's a general problem with contrast ratio of inverted ObjectStatus, then please open a ticket in central design and post the ticket ID here.
